### PR TITLE
chore: update dockerfile

### PR DESCRIPTION
**Pull request by evenstensberg**
**What kind of change does this PR introduce?**

Updating deps

**Did you add tests for your changes?**
N/A

**If relevant, did you update the documentation?**
N/A
**Summary**
Updates to node v18

**Does this PR introduce a breaking change?**

No

**Other information**
N/A

### CODE REVIEW

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM node:16
+FROM node:18
```
Comment (snitin315) [inline .devcontainer/Dockerfile:1]: 16 is still LTS

Comment (evenstensberg) [reply .devcontainer/Dockerfile:1]: Docs on this?

Comment (snitin315) [reply .devcontainer/Dockerfile:1]: https://nodejs.org/en/

**Review by rishabh3112 — CHANGES_REQUESTED** [review]
I think we need to have overhead to updating this with every node release.
I have a simple solution in mind to fix it. Will be creating a PR.

**Comment by rishabh3112** [comment]
Superseded by #3311

**Comment by rishabh3112** [comment]
Thanks

**Pull request closed without merging**